go/cmd/dolt/commands/select_where.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
```

Comment:
This stuff was moved here from the tblcmds package.  The "tblcmds" package depends on the "commands" package so any shared code needs to live in the commands package to prevent a circular dependency.